src/blackduck-security-action/bridge-cli.ts

```diff
@@ -135,9 +135,9 @@ export class Bridge {
         bridgeVersion = await this.getBridgeVersionFromLatestURL(this.bridgeArtifactoryURL.concat('latest/versions.txt'))
         bridgeUrl = this.getLatestVersionUrl()
       }
+      info('Bridge Version is - '.concat(bridgeVersion))
```

Comment:
@gouravjblackduck can you confirm in case of network airgap does it work fine?

Reply:
> Should it be "Bridge CLI" instead of "Bridge" ?
this is fixed , 

For Airgap Tithi confirmed we will not print version as we have to write code for that for now we will skip that

Reply:
Got it. Thanks